bigtable/bigtable.go

```diff
@@ -1581,6 +1581,9 @@ func gaxInvokeWithRecorder(ctx context.Context, mt *builtinMetricsTracer, method
 
 		// f makes calls to CBT service
 		err := f(ctx, &attemptHeaderMD, &attempTrailerMD, callSettings)
+		if !mt.builtInEnabled {
+			return err
+		}
```

Comment:
Why do bigtable calls fail if client side metrics is disabled?

Reply:
They don't
This is sort of short cicuiting the code. The lines after this line record metrics. If built in metrics has been disabled, we don't want to record anything.

I have refactored the code to make the behavior more obvious.